core/src/style/style.h

```diff
@@ -54,7 +54,7 @@ class Style {
     std::string m_name;
 
     /* <ShaderProgram> used to draw meshes using this style */
-    std::shared_ptr<ShaderProgram> m_shaderProgram = std::make_shared<ShaderProgram>();
+    std::unique_ptr<ShaderProgram> m_shaderProgram = std::make_unique<ShaderProgram>();
```

Comment:
`std::make_unique` which should be part of c++14 seems to be breaking the travis build, any idea why?